src/Main.elm

```diff
@@ -540,6 +539,19 @@ swapArrayIndex i j arr =
 
 view : Model -> Html Msg
 view model =
+    case model.initError of
+        Just errString ->
+            div [ class "tff-error" ]
+                [ h3 [] [ text "This form could not be initialized: " ]
+                , pre [] [ text errString ]
+                ]
+
+        Nothing ->
+            viewMain model
+
+
+viewMain : Model -> Html Msg
+viewMain model =
```

Comment:
and our `view` function will show the error 

![Screenshot 2024-09-14 at 2 35 22 PM](https://github.com/user-attachments/assets/d2875dda-0e3e-40bc-b2ec-c726041e4a50)
![Screenshot 2024-09-14 at 2 35 24 PM](https://github.com/user-attachments/assets/1c83a025-00e9-4d29-aa07-178d0ed83240)